src/fastapi_zitadel_auth/testing/utils.py

```diff
@@ -1,44 +1,47 @@
 """
-Test utilities
+Utility functions and constants for testing Zitadel authentication
 """
 
 import os
 from datetime import datetime, timedelta
+from typing import Dict, Any, Optional
 
 import jwt
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import serialization
 from cryptography.hazmat.primitives.asymmetric import rsa
 
-# Global test settings
-ZITADEL_ISSUER = os.environ["ZITADEL_HOST"]  # The Zitadel issuer URL
-ZITADEL_PROJECT_ID = os.environ["ZITADEL_PROJECT_ID"]  # The project ID where the API app is located
-ZITADEL_CLIENT_ID = os.environ["OAUTH_CLIENT_ID"]  # The client ID of the API app
-ZITADEL_PRIMARY_DOMAIN = "client-fza.region.zitadel.cloud"  # The primary domain of a Zitadel client
+from fastapi_zitadel_auth import ZitadelAuth
 
+# Test constants - can be overridden with environment variables or custom values
+ZITADEL_ISSUER = os.getenv("ZITADEL_HOST", "https://test.zitadel.cloud")
+ZITADEL_PROJECT_ID = os.getenv("ZITADEL_PROJECT_ID", "test-project-id")
+ZITADEL_CLIENT_ID = os.getenv("OAUTH_CLIENT_ID", "test-client-id")
+ZITADEL_PRIMARY_DOMAIN = os.getenv("ZITADEL_PRIMARY_DOMAIN", "test.zitadel.cloud")
```

Comment:
I'd prefer to not use defaults since we already specify them in `.env.test`: https://github.com/cleanenergyexchange/fastapi-zitadel-auth/blob/main/tests/.env.test and pyproject.toml > ini options: https://github.com/cleanenergyexchange/fastapi-zitadel-auth/blob/c4a11bc318bf84bf93c1d089377b2bd29905bc6d/pyproject.toml#L78

Reply:
This is actually somewhat of a problem; because now, when I import it it will fail because these exact environment variables are not set; the client app may have different ENVs for this (already). 

I guess we can support having one fixture to override these variables? What do you think?

Reply:
I know these can be tricky. One option could also be to import the demo project's `get_settings` to load the environment variables. But I'm also ok with using a fixture with `autouse=True` if it's simpler